examples/proxy/outputs.tf

```diff
@@ -8,7 +8,42 @@ output "lb_address" {
   description = "Load Balancer Address"
 }
 
-output "login_url" {
-  value       = module.tfe.login_url
+output "health_check_url" {
+  value       = module.tfe.health_check_url
+  description = "The URL of the Terraform Enterprise health check endpoint."
+}
+
+output "iact_url" {
+  value       = module.tfe.iact_url
+  description = "IACT URL"
+}
+
+output "initial_admin_user_url" {
+  value       = module.tfe.initial_admin_user_url
+  description = "Initial Admin user URL"
+}
+
+output "url" {
+  value       = module.tfe.url
   description = "Login URL to setup the TFE instance once it is initialized"
 }
+
+output "network" {
+  value       = module.tfe.network
+  description = "The network to which TFE is attached."
+}
+
+output "subnetwork" {
+  value       = module.tfe.subnetwork
+  description = "The subnetwork to which TFE is attached."
+}
+
+output "dns_configuration_notice" {
+  value       = "If you are using external DNS, please make sure to create a DNS record using the lb_address output that has been provided"
+  description = "A warning message."
+}
+
+output "iact_notice" {
+  value       = "Once deployed, please follow this page to set the initial user up: https://www.terraform.io/docs/enterprise/install/automating-initial-user.html"
+  description = "Login advice message."
+}
```

Comment:
None of these are really needed. However, if you choose the leave them, you'll need to fix the values on lines 42 and 47.